src/test/java/integration/DancerRepositoryIT.java

```diff
@@ -16,6 +17,7 @@
 @SpringBootTest
 @ContextConfiguration(classes = TestConfig.class)
 @ActiveProfiles("test")
+@DirtiesContext
```

Comment:
@kermakovich what is the purpose of usage DirtiesContext?

Reply:
@h1alexbel I use `neo4jContainer::getBoltUrl` in @DynamicPropertySource and without @DirtiesContext only first test class runs  correctly, but the other classes fail.

Reply:
OK